specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -2666,23 +2725,22 @@
       }
     },
     "PrivateEndpointConnectionProvisioningState": {
```

Comment:
>PrivateEndpointConnectionProvisioningState [](http://example.com/codeflow?start=5&length=42)

Does this conform to the common-types privateEndpointConnection definition? Can it be changed to a $ref? 

[https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/privatelinks.json#L20](https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/privatelinks.json#L20)

Reply:
same as above, I will confirm code behavior and will move to common in the next version if possible.